# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/PBrowser.ipdl
# Commit: d34a8ec4785c
# Full Hash: d34a8ec4785c335bfad41186b200e4ad81d772a4
# Author: Barret Rennie <barret@brennie.ca>
# Date: 2019-04-04 04:33:36
# Regressor Bug: 1510569
# File Overlap Count: 1
# Description:
#   Bug 1510569 - Include innerDOMWindowID in the WebProgressData sent from TabChild to TabParent r=baku
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D24808
# ==============================================================================

diff -r b075ab8b8be4 -r d34a8ec4785c dom/ipc/PBrowser.ipdl
--- a/dom/ipc/PBrowser.ipdl	Wed Apr 03 17:29:40 2019 +0000
+++ b/dom/ipc/PBrowser.ipdl	Wed Apr 03 17:29:58 2019 +0000
@@ -108,7 +108,8 @@
   bool isTopLevel;
   bool isLoadingDocument;
   uint32_t loadType;
-  uint64_t DOMWindowID;
+  uint64_t outerDOMWindowID;
+  uint64_t innerDOMWindowID;
 };
 
 struct RequestData